nox/sessions.py

```diff
@@ -379,6 +379,10 @@ def run(
         if not args:
             raise ValueError("At least one argument required to run().")
 
+        if len(args) == 1 and isinstance(args[0], list):
```

Comment:
```suggestion
        if len(args) == 1 and isinstance(args[0], (list, tuple)):
```

I think it might make sense to check for tuple too?